src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -198,8 +197,15 @@ public TimeSpan NonceLifetime
         /// <summary>
         /// Gets or sets a value indicating if a 'sub' claim is required.
         /// </summary>
-        [DefaultValue(false)]
-        public bool RequireSub { get; set; }
+        [DefaultValue(true)]
```

Comment:
The DefaultValueAttribute is just for designers to use to set a default value in the UI, which doesn't apply to Wilson. Why do we set the DefaultValueAttribute?